django_project/localities_osm/serializer/locality_osm_centroid.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+__author__ = 'Anita Hapsari <anita@kartoza.com>'
+__date__ = '10/04/19'
+
+from rest_framework import serializers
+from localities_osm.models.locality import LocalityOSMView
+
+
+class LocalityOSMCentroidSerializer(serializers.ModelSerializer):
+    lat = serializers.SerializerMethodField()
+    lng = serializers.SerializerMethodField()
+
+    class Meta:
+        model = LocalityOSMView
+        fields = [
+            'lat',
+            'lng'
+        ]
+
+    def get_lat(self, instance):
```

Comment:
hi @ann26 
maybe you can use to_representation function
because the field is from same source

Reply:
I changed the way it fetch the latitude and longitude of the centroid, now they are different